### PR TITLE
WLSSB Revamp

### LINKED ISSUE #26: Update Super Staff Bros.
Main TODO:

- [x] Remove former auth for SG
- [x] Update Ranks

- [x] Import Tsunami's Auth

- [x] Add  wgc (Nothing i can do rn so i just left a message on the txt file)

- [x] New Descriptions

----
Also if thou must, This is probably a good time to update any staffmons if you feel like it!

### PR DESCRIPTION
**Pull request by Lycanium-Z**
When done this closes #26

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
I started for now on the extremely recent SGSSB

File: mods/wlssb/scripts.js [hunk 1]
```diff
@@ -35,4 +35,24 @@ exports.BattleScripts = {
...
+		damage(d, source, effect) {
+			if (!this.hp) return 0;
+			if (d < 1 && d > 0) d = 1;
+			d = Math.floor(d);
+			if (isNaN(d)) return 0;
+			if (d <= 0) return 0;
+			this.hp -= d;
+			if (this.hp <= 0) {
+				d += this.hp;
+				if (this.item === 'polkadotbow' && this.species === 'Lycanroc') {
```
Comment (HoeenCoder) [inline mods/wlssb/scripts.js]: Ok @Lycanium-Z what does your mon do anyways?

Comment (Lycanium-Z) [reply mods/wlssb/scripts.js]: feelsnv

Comment (HoeenCoder) [reply mods/wlssb/scripts.js]: ... no, hell no.

I asked you simply to make a ballanced game, did you really ahve to do this? we have a deadline and more to do, no time to waste! nerf it properly, or i will myself

Comment (Lycanium-Z) [reply mods/wlssb/scripts.js]: Alright Alright will do!

**Comment by Lycanium-Z** [comment]
dealt with all issues @HoeenCoder @DeathlyPlays

**Comment by Lycanium-Z** [comment]
Will Test soon

**Comment by Lycanium-Z** [comment]
tfw HoeenHero and me finally compromised on an WLSSB mon

**Pull request merged**